extensions/wikia/CommunityHeader/tests/NavigationTest.php

```diff
@@ -43,341 +35,398 @@ public function testExploreItems( $globals, $expectedExploreItems ) {
 	 * @param $expected
 	 */
 	public function testDiscussLink( $globals, $expected ) {
-		$globals = new GlobalStateWrapper( [
-			'wgEnableDiscussions' => $globals['wgEnableDiscussions'],
-			'wgEnableForumExt' => $globals['wgEnableForumExt'],
-		] );
+		$this->mockStaticMethodWithCallBack(
+			'WikiFactory',
+			'getVarValueByName',
+			function ( $variable/*, $id*/ ) use ( $globals ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$globals" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)